docs/reference/query-languages/esql/functions-operators/aggregation-functions.md

```diff
@@ -0,0 +1,59 @@
+---
+navigation_title: "Aggregation functions"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/functions-operators/aggregation-functions.html
```

Comment:
```suggestion
  - https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-functions-operators.html#esql-agg-functions
```

This metadata maps pages from old system to new system, and they have been impacted by find/replaces elsewhere

Reply:
So the url to use for mapped_pages should be the old URL? I can make that change. I had thought I would need to add a bunch of entries to redirects, but perhaps this has the same effect?

Reply:
@craigtaverner this is purely for migration purposes and is unrelated to redirects.yml